examples/interpreter-demo/async-execution.html

```diff
@@ -43,166 +42,140 @@ <h1>Asynchronous Execution with JS-Interpreter</h1>
-                        },
-                        "inputs": {
-                          "A": {
-                            "block": {
-                              "type": "variables_get",
-                              "fields": {
-                                "VAR": {
-                                  "id": ",4tsL~{roFQW!-CL1(Gn"
-                                }
-                              }
-                            }
-                          },
-                          "B": {
-                            "block": {
-                              "type": "math_number",
-                              "fields": {
-                                "NUM": 2
-                              }
-                            }
-                          }
-                        }
-                      }
-                    }
-                  },
-                  "next": {
-                    "block": {
-                      "type": "text_print",
-                      "id": "print",
-                      "inputs": {
-                        "TEXT": {
-                          "block": {
-                            "type": "variables_get",
-                            "fields": {
-                              "VAR": {
-                                "id": ",4tsL~{roFQW!-CL1(Gn"
-                              }
-                            }
-                          }
-                        }
-                      }
-                    }
-                  }
-                }
-              }
-            }
-          }
-        }
-      },
-      {
-        "type": "variables_set",
-        "x": 0,
-        "y": 0,
-        "fields": {
-          "VAR": {
-            "id": ",4tsL~{roFQW!-CL1(Gn"
-          }
-        },
-        "inputs": {
-          "VALUE": {
-            "block": {
-              "type": "math_number",
-              "fields": {
-                "NUM": 1
-              }
-            }
-          }
-        },
-        "next": {
-          "block": {
-            "type": "controls_repeat_ext",
-            "inputs": {
-              "TIMES": {
-                "block": {
-                  "type": "math_number",
-                  "fields": {
-                    "NUM": 4
-                  }
-                }
-              }
-            }
-          }
-        }
-      }
-    ]
-  },
-  "variables": [
-    {
-      "name": "n",
-      "id": ",4tsL~{roFQW!-CL1(Gn"
-    }
-  ]
-}
     var demoWorkspace = Blockly.inject('blocklyDiv',
         {media: 'https://unpkg.com/blockly/media/',
-         toolbox: toolboxJson});
+         toolbox: document.getElementById('toolbox')});
     Blockly.JavaScript.STATEMENT_PREFIX = 'highlightBlock(%1);\n';
     Blockly.JavaScript.addReservedWords('highlightBlock');
-    Blockly.serialization.workspaces.load(startBlocks, demoWorkspace)
-
-
-    // Exit is used to signal the end of a script.
-    Blockly.JavaScript.addReservedWords('exit');
```

Comment:
I have absolutely no idea what 'exit' is about.  AnmAtAnm added it in 2018 for unknown reasons.